app/admin/allocations.rb

```diff
@@ -1,36 +1,25 @@
 # frozen_string_literal: true
 
 ActiveAdmin.register Allocation do
-  config.sort_order = ''
+  config.sort_order = 'ongoing_desc'
   permit_params :user_id, :project_id, :start_at, :end_at, :company_id, :ongoing
 
   config.batch_actions = false
 
   menu parent: User.model_name.human(count: 2), priority: 4
 
-  scope :ongoing, default: true
-  scope :finished
-  scope :all do
-    current_user.company.allocations.all.joins(:user).merge(User.active)
-  end
-  scope :spreadsheet do |relation|
-    AllocationsAndUnalocatedUsersQuery.new(relation, current_user.company).call
-  end
-
+  filter :ongoing
   filter :user, collection: proc {
-    current_user.super_admin? ? User.all.order(:name).group_by(&:company) : current_user.company.users.order(:name)
```

Comment:
just curious, why do we want to remove this `group_by`?

Reply:
Because it causes a bug when the user accessing the page is a super admin. If the dropdown has subgroups of companies, it generates options like this one: `<option value="name">name</option>`.